packages/opencensus-web-exporter-ocagent/package.json

```diff
@@ -43,7 +42,7 @@
     "access": "public"
   },
   "devDependencies": {
-    "@opencensus/core": "^0.0.8",
+    "@opencensus/core": "0.0.9",
```

Comment:
Here we are using "pin" dependency, but using semver range in ```opencensus-web-core``` package. I think we should be consistent across. WDYT?

Reply:
Good point - this was just a mistake. Changed it to the `^0.0.9` semver range to match `opencensus-web-core` and the other dependencies.